sdks/unreal/Agones/Source/Agones/Private/Agones.cpp

```diff
@@ -1,92 +1,18 @@
-// Copyright 2019 Google LLC All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+// Copyright Epic Games, Inc. All Rights Reserved.

```

Comment:
Unfortunately I don't think we can't accept this with the current copyright :disappointed:  not sure if that is going to be an issue?

Reply:
This was due to taking the short cut of generating via UE4 can rewrite exactly the same by hand.